akka-persistence/src/main/scala/akka/persistence/Eventsourced.scala

```diff
@@ -350,7 +350,7 @@ private[persistence] trait Eventsourced extends Snapshotter with Stash with Stas
    * @param events events to be persisted
    * @param handler handler for each persisted `events`
    */
-  final def persistAllAsync[A](events: immutable.Seq[A])(handler: A ⇒ Unit): Unit = {
+  def persistAllAsync[A](events: immutable.Seq[A])(handler: A ⇒ Unit): Unit = {
     events.foreach { event ⇒
       pendingInvocations addLast AsyncHandlerInvocation(event, handler.asInstanceOf[Any ⇒ Unit])
     }
```

Comment:
This change forgot to include: 
- `persistAsync` on line 361
- `deferAsync` on line 381